test/check_binary_symbols.py

```diff
@@ -78,10 +78,14 @@ def check_lib_symbols_for_abi_correctness(lib: str, pre_cxx11_abi: bool = True)
             raise RuntimeError("Didn't find enought cxx11 symbols")
 
 def main() -> None:
-    if os.getenv("PACKAGE_TYPE") == "libtorch":
-       install_root = Path(os.getcwd())
+    if "install_root" in os.environ:
```

Comment:
install_root is set in check_binary already, hence if its set - no need to figure it out.